java/se/sics/mspsim/core/RTC.java

```diff
@@ -379,7 +379,7 @@ private void rtcInit() {
          * Reset the calendar to 01.01.0000
          */
         private void resetCalendar() {
-                cal = new GregorianCalendar(0, 1, 1, 0, 0, 0);
+                cal = new GregorianCalendar(0, Calendar.FEBRUARY, 1, 0, 0, 0);
```

Comment:
I think this looks like a typo in the original code and it should be Calendar.JANUARY since the comment says reset to "01.01.0000". Same for line 244.

Reply:
Fixed.